registration-processor/workflow-engine/registration-processor-workflow-manager-service/src/main/resources/bootstrap.properties

```diff
@@ -1,17 +1,23 @@
 spring.cloud.config.uri=http://localhost:51000/config
 spring.cloud.config.label=master
 spring.application.name=application,registration-processor
-spring.profiles.active=mz
+spring.profiles.active=default
```

Comment:
Please change to spring.profiles.active=mz

Reply:
done ,please check earliest commit